frontend/e-buddy/src/app/components/FileItem.tsx

```
@@ -0,0 +1,52 @@
+  return (
+    <li key={file.name} className="bg-gray-50 p-4 rounded-lg shadow-sm flex flex-col">
+      <div className="flex items-center justify-between">
+        <div className="flex items-center">
+          {getFileIcon(file.name)}
+          <span className="font-medium text-gray-700">{file.name}</span>
+        </div>
+        <button
+          onClick={() => onRemove(file)}
+          className="text-red-500 hover:text-red-700"
+          aria-label={`Remove ${file.name}`}
+        >
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)